packages/dev/core/src/AudioV2/webAudio/subProperties/spatialWebAudioListener.ts

```diff
@@ -18,109 +19,85 @@ export function _CreateSpatialAudioListener(engine: _WebAudioEngine, autoUpdate:
-        this._rotationQuaternion = value;
-        this._updateRotation();
+    /** @internal */
+    public set minUpdateTime(value: number) {
+        this._updaterComponent.minUpdateTime = value;
     }
 
     /** @internal */
     public override dispose(): void {
         super.dispose();
 
-        this._autoUpdate = false;
+        this._updaterComponent.dispose();
+        this._updaterComponent = null!;
     }
 
     /** @internal */
     public update(): void {
-        this._updatePosition();
-        this._updateRotation();
+        if (this.isAttached) {
+            this._attacherComponent?.update();
+        } else {
+            this.updatePosition();
+            this.updateRotation();
+        }
     }
 
-    private _updatePosition(): void {
-        if (this._lastPosition.equalsWithEpsilon(this._position)) {
+    /** @internal */
+    public updatePosition(): void {
+        if (this._lastPosition.equalsWithEpsilon(this.position)) {
             return;
         }
 
         const listener = this._audioContext.listener;
-        listener.positionX.value = this._position.x;
-        listener.positionY.value = this._position.y;
-        listener.positionZ.value = this._position.z;
+        listener.positionX.value = this.position.x;
+        listener.positionY.value = this.position.y;
+        listener.positionZ.value = this.position.z;
 
-        this._lastPosition.copyFrom(this._position);
+        this._lastPosition.copyFrom(this.position);
     }
 
-    private _updateRotation(): void {
-        if (!this._lastRotationQuaternion.equalsWithEpsilon(this._rotationQuaternion)) {
-            TmpQuaternion.copyFrom(this._rotationQuaternion);
-            this._lastRotationQuaternion.copyFrom(this._rotationQuaternion);
-        } else if (!this._lastRotation.equalsWithEpsilon(this._rotation)) {
-            Quaternion.FromEulerAnglesToRef(this._rotation.x, this._rotation.y, this._rotation.z, TmpQuaternion);
-            this._lastRotation.copyFrom(this._rotation);
+    /** @internal */
+    public updateRotation(): void {
+        if (!this._lastRotationQuaternion.equalsWithEpsilon(this.rotationQuaternion)) {
+            TmpQuaternion.copyFrom(this.rotationQuaternion);
+            this._lastRotationQuaternion.copyFrom(this.rotationQuaternion);
+        } else if (!this._lastRotation.equalsWithEpsilon(this.rotation)) {
+            Quaternion.FromEulerAnglesToRef(this.rotation.x, this.rotation.y, this.rotation.z, TmpQuaternion);
+            this._lastRotation.copyFrom(this.rotation);
         } else {
             return;
         }
 
         Matrix.FromQuaternionToRef(TmpQuaternion, TmpMatrix);
         const listener = this._audioContext.listener;
 
-        Vector3.TransformNormalToRef(Vector3.Forward(), TmpMatrix, TmpVector);
+        // Use `Vector3.Backward()` instead of `Forward()` because the WebAudio API is right-handed.
+        Vector3.TransformNormalToRef(Vector3.Backward(), TmpMatrix, TmpVector);
```

Comment:
```suggestion
        Vector3.TransformNormalToRef(Vector3.RightHandedBackwardReadOnly, TmpMatrix, TmpVector);
```

Also, a small note - both Forward and backward have an optional rightHanded parameter. But it is better to use the read-only here anyhow.